File: C++/DataAnalysys/Book.cpp
#include "Book.h"
#include <iostream>
#include <fstream>
#include <string>
using namespace std;

Book::Book(int _id, string _author, int _year_published, string _title, double _rating, string _is_issued) {
	id = _id;
	author = _author;
	year_published = _year_published;
	title = _title;
	rating = _rating;
	is_issued = _is_issued;
}

File: Book.py
class Book:
    """ Class representing a book in the system"""
    def __init__(self, id, author, publication_year, title, rating, is_issued=False):
        self.id = id
        self.author = author
        self.publication_year = publication_year
        self.title = title
        self.rating = rating
        self.is_issued = is_issued

    def print_details(self):
        """ Method to print the details about the book"""
        print("{} {} {} {} {}".format(self.id, self.author, self.publication_year, self.title, self.rating))
